release-notes.txt

```diff
@@ -1,3 +1,780 @@
+RIOT-2020.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2020.07 release puts a focus on the improvement of automated testing. In
+that vain various new CI integrations such as CircleCI for documentation
+building and online presentation and Github Actions to check the tooling of
+RIOT were merged. The `riotctrl_shell` python module allows to abstract shell
```

Comment:
```suggestion
RIOT when merged. The `riotctrl_shell` python module allows to abstract shell
```

Reply:
Addressed and squashed directly.

Reply:
Now that I'm reading this again, the initial word `were` was ok.